docs/integration/categories/network_security/zscaler_zia.md

```diff
@@ -95,6 +97,49 @@ In the Zscaler ZIA console:
+		```
+
+	=== "SaaS Security Activity"
+
+		```
+		<PRI>1 %d{yyyy}-%02d{mm}-%02d{dd}T%02d{hh}:%02d{mm}:%02d{ss}Z zscaler-nss NSS - - - event=login="%s{username}" tenant="%s{tenant}" object_type="%d{objtype1}" applicationname="%s{appname}" object_name_1="%s{objnames1}" object_name_2="%s{objnames2}"
+		```
+
+	=== "Tunnel IKE Phase 1"
+		```
+		<PRI>1 %d{yyyy}-%02d{mm}-%02d{dd}T%02d{hh}:%02d{mm}:%02d{ss}Z zscaler-nss NSS - - - event=datetime="%s{datetime}" Recordtype="%s{tunnelactionname}" tunneltype="IPSEC IKEV %d{ikeversion}" user="%s{vpncredentialname}" location="%s{elocationname}" sourceip="%s{sourceip}" destinationip="%s{destvip}" sourceport="%d{srcport}" destinationport="%d{dstport}" lifetime="%d{lifetime}" ikeversion="%d{ikeversion}" spi_in="%lu{spi_in}" spi_out="%lu{spi_out}" algo="%s{algo}" authentication="%s{authentication}" authtype="%s{authtype}" recordid="%d{recordid}"
+		```
+
+	=== "Tunnel IKE Phase 2"
+
+		```
+		<PRI>1 %d{yyyy}-%02d{mm}-%02d{dd}T%02d{hh}:%02d{mm}:%02d{ss}Z zscaler-nss NSS - - - event=datetime="%s{datetime}" Recordtype="%s{tunnelactionname}" tunneltype="IPSEC IKEV %d{ikeversion}" user="%s{vpncredentialname}" location="%s{elocationname}" sourceip="%s{sourceip}" destinationip="%s{destvip}" sourceport="%d{srcport}" sourceportstart="%d{srcportstart}" destinationportstart="%d{destportstart}" srcipstart="%s{srcipstart}" srcipend="%s{srcipend}" destinationipstart="%s{destipstart}" destinationipend="%s{destipend}" lifetime="%d{lifetime}" ikeversion="%d{ikeversion}" lifebytes="%d{lifebytes}" spi="%d{spi}" algo="%s{algo}" authentication="%s{authentication}" authtype="%s{authtype}" protocol="%s{protocol}" tunnelprotocol="%s{tunnelprotocol}" policydirection="%s{policydirection}" recordid="%d{recordid}"
+		```
+
+	=== "Tunnel"
+
+		```
+		<PRI>1 %d{yyyy}-%02d{mm}-%02d{dd}T%02d{hh}:%02d{mm}:%02d{ss}Z zscaler-nss NSS - - - event=datetime="%s{datetime}" Recordtype="%s{tunnelactionname}" tunneltype="%s{tunneltype}" user="%s{vpncredentialname}"
+		```
+
+	=== "Admin Audit"
+
+		```
+		<PRI>1 %d{yyyy}-%02d{mth}-%02d{dd}T%02d{hh}:%02d{mm}:%02d{ss}Z zscaler-nss NSS - [event time="%s{time}" recordid="%d{recordid}" action="%s{action}" category="%s{category}" subcategory="%s{subcategory}" resource="%s{resource}" interface="%s{interface}" adminid="%s{adminid}" clientip="%s{clientip}" result="%s{result}" errorcode="%s{errorcode}" auditlogtype="%s{auditlogtype}" preaction="%s{epreaction}" postaction="%s{epostaction}"]
```

Comment:
Why the payloads of the syslog message are not JSON-document template? The [format](https://github.com/SEKOIA-IO/intake-formats/tree/main/Zscaler/zscaler-zia) only accepts JSON formatted events.
Can you fix them? i.e, copy the payload from the RFC3164 formats?
Can you also replace `<PRI>` by `<134>`? I expect some customers putting `<PRI>` in their template (that will generate invalid syslog messages).